src/overlays/actors/ovl_Obj_Kibako/z_obj_kibako.h

```diff
@@ -3,13 +3,28 @@
 
 #include <global.h>
 
+#define GET_KIBAKO_COLLECTIBLE_ID(x) (((x->actor.params >> 0x8) & 0x7F) << 8)
+#define D_05007890 0x05007890
+#define D_05007980 0x05007980
+#define D_06001180 0x06001180
+#define D_06001A70 0x06001A70
```

Comment:
Instead of doing this, you should add those symbols to `linker_scripts/object_script.txt`.